### ISSUE TITLE
Question about the system

### ISSUE THREAD

**Issue by ugintl** [PadamSethia/noice#6]
If I develop it locally and upload it to a shared hosting, will it work? Does my hosting need to have python support?

**Comment by highoncarbs** [comment]
Shared hosting - you'll have to install python . All shared hosting have the ability to run Pyhton

**Comment by xXWarMachineRoXx** [comment]
I feel that the requirements.py is not up to date.

would you be willing to merge a updated requirements.txt , if I send a PR @highoncarbs ?

**Comment by highoncarbs** [comment]
@xXWarMachineRoXx  Sure!

**Comment by xXWarMachineRoXx** [comment]
Hi @highoncarbs 👋 ,

Thanks for the prompt reply.

Update: 
I actually successfully got this project to work with latest python packages
Can I commit the whole thing too?

**Comment by highoncarbs** [comment]
Yep, also please share how are you using this. Maybe if it has more traction, I can re-start this repo with updates
